packages/compass-generative-ai/src/atlas-ai-service.ts

```diff
@@ -395,6 +467,60 @@ export class AtlasAiService {
     );
   }
 
+  /**
+   * @returns {MockDataSchemaResponse} which contains faker.js mappings used to produce a
+   * faker.js factory function for the purposes of generating mock document data.
```

Comment:
This is incorrect, you are returning a promise of a response, not a response, it's also redundant to describe types through jsdoc when you already have types in typescript, so you can just remove this

```suggestion
   * @returns response which contains faker.js mappings used to produce a
   * faker.js factory function for the purposes of generating mock document data.
```